docs-client/src/containers/MethodPage/index.tsx

```diff
@@ -210,6 +237,63 @@ export default class MethodPage extends React.PureComponent<Props, State> {
                     className: 'code',
                   }}
                 />
+                {isAnnotatedHttpService &&
+                  ((isExactPathMapping(method) && (
+                    <>
+                      <Typography variant="body1" paragraph />
+                      <Button
+                        color="secondary"
+                        onClick={this.onAddHttpQueriesClick}
+                      >
+                        Add HTTP queries
+                      </Button>
+                      <Typography variant="body1" paragraph />
+                      {this.state.additionalQueriesOpen && (
+                        <>
+                          <TextField
+                            multiline
+                            fullWidth
+                            rows={1}
+                            value={this.state.additionalQueries}
+                            onChange={this.onQueriesFormChange}
+                            inputProps={{
+                              className: 'code',
+                            }}
+                          />
+                        </>
+                      )}
+                    </>
+                  )) ||
+                    (!isExactPathMapping(method) && (
+                      <>
+                        <Typography variant="body1" paragraph />
+                        <Button
+                          color="secondary"
+                          onClick={this.onInsertEndpointPathClick}
+                        >
+                          Insert the endpoint path
```

Comment:
`Endpoint path`?

Reply:
How about also moving this form before the first text area?

Reply:
Fixed and Moved. Thank you~!